browse/templates/feedback_collector_js.html

```diff
@@ -8,8 +8,8 @@
     });
   },
   fieldValues: {
-    "components": ["15700"],  // Browse component.
-    "versions": ["14132"],  // Release browse-0.1
+    "components": ["15700"],  // browse component
```

Comment:
These are "magic numbers" from JIRA, right? might be worth highlighting where they came from.